themes/foulab-2018/layouts/shortcodes/centered-image.html

```diff
@@ -1,3 +1,7 @@
-<CENTER>
-  <img src="{{ .Site.BaseURL }}/media/image/{{ .Page.File.Dir }}{{ .Page.File.BaseFileName }}/{{ .Get "src" }}" {{ with .Get "height" }}height="{{ . }}"{{ end }} {{ with .Get "width" }}width="{{ . }}"{{ end }} />
-</CENTER>
+<img
+  src="/media/image/{{ replace .Page.File.Dir "\\" "/"}}{{ .Page.File.BaseFileName }}/{{ .Get "src" }}"
```

Comment:
What's the reason for this 'replace'? Is it because you were developing on Windows and it was necessary to construct a valid path?

Reply:
Actually yes, there was some weirdness with the Dir having a backslash inside of it - so I was getting a path that was `%5c` instead of forward slashes.